### PR TITLE
Version # and README update

### PR DESCRIPTION
**Pull request by langsmith**
Addresses @zugaldia 's requests:  https://github.com/mapbox/mapbox-navigation-android/issues/4#issuecomment-295871541

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,27 @@
 # mapbox-navigation-android
```
Comment (zugaldia) [inline README.md]: This line can be removed.

File: README.md [hunk 2]
```diff
@@ -1,31 +1,27 @@
...
 
+```
+// Mapbox Navigation SDK for Android
+compile('com.mapbox.mapboxsdk:mapbox-android-navigation:0.1.0@aar') {
+    transitive = true
+}
+```
 
 
 ### Using the Navigation SDK
```
Comment (zugaldia) [inline README.md]: Let's add a link to the Test App in this repo for examples, and to https://www.mapbox.com/android-docs/ -> navigation.

File: README.md [hunk 3]
```diff
@@ -1,31 +1,27 @@
...
-
-<!---
-Keep adding text...
--->
-
-[Our full documentation on Mapbox Navigation](https://www.mapbox.com/android-docs/mapbox-navigation/2.1.0/navigation/)
+The Mapbox Navigation SDK for Android is built on top of [the Mapbox Directions API](https://github.com/mapbox/mapbox-java/blob/master/mapbox/libjava-services/src/main/java/com/mapbox/services/api/directions/v5/DirectionsService.java) and contains logic needed to get timed navigation instructions. We are designing new maps specifically for navigation that highlight traffic conditions and helpful landmarks. The calculations use the user's current location and compare it to the current route that the user's traversing to provide critical information at any given moment. _You control the entire experience, from the time your user chooses a destination to when they arrive._
```
Comment (zugaldia) [inline README.md]: Let's link to https://www.mapbox.com/directions instead.

**Comment by cammace** [comment]
Noting that this readme should probably follow the same format we use in the [`mapbox-java` readme](https://github.com/mapbox/mapbox-java/blob/master/README.md).

**Pull request merged**